client/src/components/leftPanel/EditPortfolios.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom'
+
+import { createPortfolio, deletePortfolio } from '../../utils/serverApi'
+import deleteIcon from '../../assets/delete.svg'
+import './EditPortfolios.scss'
+import './TagEdit.scss'
+import Modal from './Modal'
+import Loader from '../ui/Loader'
+
+class EditPortfolios extends Component {
+  state = {
+    addInput: '',
+    deleteSearchInput: '',
+    loading: false,
+    resultType: ''
+  }
+
+  onClose = () => {
+    this.props.history.push(`/`)
+  }
+
+  onAddInputChange = (e) => {
+    this.setState({resultType: ''}) // reset success/error
+    this.setState({addInput: e.target.value})
+  }
+
+  onCreate = async (e, name) => {
+    e.preventDefault() // prevent reloading page with button click
```

Comment:
I believe this is default behaviour on `<input type="submit" />`, not on `<button />` element.

Reply:
Therefore you can remove this line, as it has no effect for <button /> tag (they don't reload the page)